src/main/java/uk/gov/companieshouse/api/testdata/service/impl/TestDataServiceImpl.java

```diff
@@ -450,35 +440,6 @@ public boolean deleteUserData(String userId) {
         return this.userService.delete(userId);
     }
 
-    @Override
-    public IdentityData createIdentityData(IdentitySpec identitySpec) throws DataException {
-        if (identitySpec.getUserId() == null || identitySpec.getUserId().isEmpty()) {
```

Comment:
I thought the pattern was that everything came through this service so why has it been removed here?  I don't particularly like it all in this one service TBH as it seems like unnecessary overhead to have this delegating out to other services - but jut checking on why that's all